quodlibet/quodlibet/ext/events/randomalbum.py

```diff
@@ -233,3 +233,10 @@ def unpause(self):
             app.player.next()
         except AttributeError:
             app.player.paused = True
+
+    def disabled_for_browser(self, browser):
+        if not browser.can_filter_albums():
```

Comment:
Surely `return browser == 'Playlists' or not browser.can_filter_albums()` is neater?